### ISSUE TITLE
separate redis test from the unit test

### ISSUE THREAD

**Issue by jchauncey** [deis/logger#99]
Since we are having to start an external component we are mixing terms when we call the test suit unit tests. Therefore, we should move the redis tests out to an integration target.

**Comment by arschles** [comment]
The separation is already done via the `testredis` build tag. The Makefile enables these tests in its `test-unit` target (https://github.com/deis/logger/blob/master/Makefile#L119), but the new build target is not in place

**Comment by jchauncey** [comment]
Yeah but that tag says to run the redis tests. I am saying dont do that as part of test-unit. Instead do that as test-integration

**Comment by arschles** [comment]
Sorry, that tag is poorly named. It _enables_ the redis tests, which run in addition to all the other tests. Thoughts on these test targets?

```Make
test-all:
  // runs all tests, including integration
  $(DOCKER_CMD) go build -tags="testredis" $(glide nv)
test-unit:
  // runs only unit tests, with no external dependencies
  $(DOCKER_CMD) go build $(glide nv)
```

**Comment by jchauncey** [comment]
yeah. i would love to be able to just run the integration tests though.

**Comment by jchauncey** [comment]
```
test-all: test-unit test-integration
```

Is what im thinking we should do

**Comment by arschles** [comment]
@jchauncey we could add capability to run just integration tests by checking a `LOGGER_TEST_INTEGRATION` env var in unit tests. If it exists, unit tests should bail out. I can research some other options too.

**Comment by Cryptophobia** [comment]
This issue was moved to teamhephy/logger#6
